src/models/disclaimer-model/resolve/disclaimers.js

```diff
@@ -24,7 +26,22 @@ export default [
       }
 
       const qcy = layoutService.getHyperCubeValue('qSize.qcy', 0);
-      return qcy > 1000;
+      return qcy > NUMBERS.MAX_NR_SCATTER && (!layoutService.meta.isBigData || !flags.isEnabled('DATA_BINNING'));
+    },
+  },
+  {
+    key: 'OverviewData',
+    alignment: 'bottom',
+    condition: ({ layoutService, flags }) => {
+      if (
+        layoutService.getLayout().showDisclaimer === false &&
+        flags.isEnabled('SHOW_DISCLAIMER') &&
```

Comment:
You don't have to look at the `SHOW_DISCLAIMER` flag (it's been enabled everywhere for a long time).